tests/test_more.py

```diff
@@ -128,6 +134,19 @@ def f():
             [['A', 'B', 'C'], ['D', 'E', 'F']],
         )
 
+    def test_strict_being_true_with_size_none(self):
+        """Test when ``n`` has value ``None`` and the keyword strict is True
+        (raising an exception).
+
+        """
+
+        def f():
+            return list(mi.chunked('ABCDE', None, strict=True))
+
+        self.assertRaisesRegex(
```

Comment:
Does this actually do what we expect? Is `f()` called?

Reply:
Oups, thanks for the review! With this mistake, I have revealed that I didn't actually write the test before writing the code, in true TDD development 😜
With my new commit, it's working as it should!